benchmark/results/csv/simple-object to string.csv

```diff
@@ -1,8 +1,9 @@
 "name","ops","margin","percentSlower"
-"devalue",1937206,1.43,60.16
-"flatted",2859492,0.31,41.19
-"next-json",2068500,0.25,57.46
-"o-son",2471220,1.8,49.18
-"seroval",3415205,0.31,29.76
-"superjson",1385356,1.23,71.51
-"warp10",4862507,8.29,0
\ No newline at end of file
+"devalue",1942992,1.39,46.95
+"flatted",3064783,0.96,16.32
+"next-json",1930111,0.76,47.3
+"o-son",3662355,0.97,0
+"seroval",2895279,1.97,20.94
+"superjson",1512774,2.01,58.69
+"turbo-stream",169238,23.26,95.38
```

Comment:
is it actually this slow??

Reply:
It was like that on my machine 😅

Reply:
I guess so. I'll merge this and test it on my own. Thanks!